python-template/{{cookiecutter.placeholder_repo_name}}/README.rst

```diff
@@ -68,11 +63,53 @@ Every time you develop something in this repo
   make validate
 
   # Commit all your changes
-  git commit …
+  git commit ...
   git push
 
   # Open a PR and ask for review.
 
+Deploying
+=========
+
+How can a new user, go about deploying this component? Is it just a few commands? Is there a larger how-to that should be linked here?
+
+For details on how to deploy this component, checkout the `deployment how-to`_
+
+.. _deployment how-to: https://docs.openedx.org/projects/this-project/how-tos/how-to-deploy-this-component.html
+
+Getting Help
+************
+
+Documentation
+=============
+
+Start by going through `the documentation`_.  If you need more help see below.
+
+.. _the documentation: https://docs.openedx.org/projects/{{cookiecutter.repo_name}}
+
+(TODO: `Set up documentation <https://openedx.atlassian.net/wiki/spaces/DOC/pages/21627535/Publish+Documentation+on+Read+the+Docs>`_)
+
+More Help
+=========
+If you're having trouble, we have discussion forums at
+https://discuss.openedx.org where you can connect with others in the
+community.
+
+Our real-time conversations are on Slack. You can request a `Slack
+invitation`_, then join our `community Slack workspace`_.
+
```

Comment:
[consider] Consider replacing with a brief message and permanent link to a doc with these details so future changes could take place in a single file.

Reply:
I think this is also a sample and different people might want to customize this as they figure out how they want this to work so I don't think there is a specific version of this that would be true for everyone.